modules/integration-browser/Readme.md

```diff
@@ -1,12 +1,41 @@
 # AWS Encryption SDK for Javascript Browser Integration
 
-This module includes compatibility tests with the other versions of the AWS Encryption SDK.
-It's designed to facilitate testing for the set of test vectors of the AWS Encryption SDK. 
-The test vectors can be found [here] (https://github.com/awslabs/aws-encryption-sdk-test-vectors/tree/a28851a188163e45b8cbf94c1d5a1e67e9622aa8).
+This repository is for compatibility tests with the other versions of the AWS Encryption SDKs.
+It's purpose is to facilitate testing the set of test vectors the AWS Encryption SDK.
```

Comment:
Typo: `Its purpose is to test the known-good static test vectors of the AWS Encryption SDK.`